data/json/bionics.json

```diff
@@ -849,7 +845,7 @@
     "name": { "str": "Air Filtration System" },
     "description": "Surgically implanted in your trachea is an advanced filtration system.  If toxins, or airborne diseases find their way into your windpipe, the filter will attempt to remove them.",
     "occupied_bodyparts": [ [ "torso", 4 ], [ "mouth", 2 ] ],
```

Comment:
```suggestion
    "occupied_bodyparts": [ [ "torso", 5 ], [ "mouth", 1 ] ],
```

Reply:
It's a filter in your respiratory system. You can put some parts outside but 2 slots make sense I think.

Reply:
Most of the trachea is in the torso. None of it is in the head, as opposed to the neck, BTW. And I'm pretty sure a filtration system wouldn't be any higher than the trachea - otherwise, it would have to deal with food coming down, instead of letting the epiglottis shield the trachea from that.